h/presenters/group_json.py

```diff
@@ -27,7 +27,7 @@ def asdict(self, expand=None):
             ),
             "public": self.group.is_public,
             # DEPRECATED: TODO: remove from client
-            "scoped": True if self.group.scopes else False,
+            "scoped": bool(self.group.scopes),
```

Comment:
I actually once had a guy introduce a function to prevent "uncontrolled uses of if". Never write an if statement again! Save time, money, and impress your friends!